### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD001.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#5] [General]
# https://blog.3place.cn/3place/fsd/FSD001.html

https://blog.3place.cn/3place/fsd/FSD001.html

**Comment by picklecai** [comment]
眼前的一切，仿佛已跟我远离，
消逝的一切，却又在化为现实。

好悲伤呀！
既被世界抛弃，又被命运掌控。
当你要真实感，真实世界远去了。
当你要自由感，枷锁桎梏又都回来了。

**Comment by picklecai** [comment]
在你们四周荡漾的魅惑的气息，
使我胸中震撼着青春的活力。

人，无论男女，在任何时候，受到魅惑，就能忘掉当下一切，荷尔蒙满满。所以有贾瑞，有“男人至死是少年”，有“你就是娘娘”……

之所以叫它魅惑，是因为只要它撤了，“满血复活”的现象也就跟着消失了。

当浪潮退去时才知道谁在裸泳。

薛宝钗说：好风凭借力，送我上青云。风停了，才知道是风筝还是飞鸟。

这是一首魅惑的诗。所以，首先得有浪潮、有风、有“娘娘”……
